openedx/core/djangoapps/course_apps/migrations/0002_initialise_course_apps.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 2.2.24 on 2021-08-11 07:10
+
+from django.db import migrations
+from opaque_keys.edx.locator import LibraryLocator
+
+from openedx.core.djangoapps.course_apps.plugins import CourseAppsPluginManager
+
+
+def initialise_course_apps(apps, schema_editor):
```

Comment:
Just building my consensus on this one. 
Will this migration run automatically with `migrate` command? 

Also given the answer is yes -- Do you this will eliminate the the possibility of "ObjectDoesNotExist" when we try to read from data model?

Reply:
We still need that code because a course apps are plugins, and a user might install a plugin in the future, after this migration is run, and then its data will be missing. We cannot rerun the migration, so in those cases we will need to use this mechanism. Additionally, in the future the platform may itself add another course app.